lib/check-or-update-routes.js

```diff
@@ -136,17 +136,22 @@ async function checkOrUpdateRoutes (options) {
   })
 
   console.log(`ℹ️  Updating routes index files`)
-  const routesByScope = allEndpoints.reduce((map, endpoint) => {
-    const [scope] = endpoint.documentationUrl.substr(state.baseUrl.length).split('/')
-    const scopeNormalized = camelCase(scope)
-    if (!map[scopeNormalized]) {
-      map[scopeNormalized] = []
-    }
-
-    map[scopeNormalized].push(endpoint)
 
-    return map
-  }, {})
+  const routesByScope = allEndpoints
+    .map(endpoint => {
+      endpoint.scope = camelCase(endpoint.documentationUrl.substr(state.baseUrl.length).split('/')[0])
```

Comment:
did you intend to add the `scope` property to the endpoint object? That’s what currently makes the tests fail. I’ll remove it for now in order to ship it, we can do a follow up PR if needed